### ISSUE TITLE
os-unifi7 since upgrading to OPNSense 2.4.1

### ISSUE THREAD

**Issue by clumsyzombie** [mimugmail/opn-repo#199]
I am just going to report this here I guess. Really green and new to OPNSense but I have been playing for about and week today I upgrade to 2.4.1 and looks like it broke unifi 7
***GOT REQUEST TO INSTALL***
Currently running OPNsense 24.1 at Tue Jan 30 18:16:34 CST 2024
Updating OPNsense repository catalogue...
OPNsense repository is up to date.
Updating mimugmail repository catalogue...
mimugmail repository is up to date.
All repositories are up to date.
pkg: libevent has a missing dependency: openssl111
pkg: mongodb44 has a missing dependency: openssl111
Checking integrity... done (0 conflicting)
Nothing to do.
***DONE***

So it wouldn't start so I tried to reload it on my testing playground and I get this error. 
Also where you thinking about upgrading to unifi8 at all?

**Comment by mimugmail** [comment]
Yep, currently building new ports with openssl, will take a day or so

**Comment by mimugmail** [comment]
https://www.reddit.com/r/opnsense/comments/1aga8ty/community_repo_update_compiled_against_openssl_to/
